ui/src/static/env.js

```diff
@@ -1,8 +1,8 @@
 window.env = {
   APPLICATION_NAME: 'Taskcluster',
   TASKCLUSTER_ROOT_URL: 'https://taskcluster.net',
-  GRAPHQL_ENDPOINT: 'http://localhost:5080/graphql',
-  GRAPHQL_SUBSCRIPTION_ENDPOINT: 'ws://localhost:5080/subscription',
+  GRAPHQL_ENDPOINT: '/graphql',
+  GRAPHQL_SUBSCRIPTION_ENDPOINT: '/subscription',
   DOCS_ONLY: 'false',
   UI_LOGIN_STRATEGY_NAMES: 'github mozilla-auth0',
```

Comment:
..and I think we would want this empty by default?

Reply:
Sure.